lang/po/Firmware_nl.po

```diff
@@ -2624,6 +2617,16 @@ msgstr "Mondstuk is heet! Wacht op afkoeling."
 msgid "Nozzle changed?"
 msgstr "Mondstuk veranderd?"
 
+#. MSG_MISSING_FILAMENT_CONTINUE c=20 r=3
+#: ../../Firmware/messages.cpp:157 ../../Firmware/util.cpp:388
+msgid "There is no filament loaded. Continue?"
+msgstr "Geen filament geladen. Doorgaan?"
+
+#. MSG_MISSING_FILAMENT_CANCELLED c=20 r=8
+#: ../../Firmware/messages.cpp:158 ../../Firmware/util.cpp:389
+msgid "There is no filament loaded. Print cancelled."
+msgstr "Geen filament geladen. Afdrukken geannuleerd."
```

Comment:
Although 'afdrukken' is the correct translation for 'to print', in the context of 3D printing it is a bit strange. I suggest to keep the English verb "to print" as that is more commonly used in the community.

So the translation should then be "Geen filament geladen. Printen geannuleerd."

With that said, other translations in this file should be adjusted accordingly for consistency's sake.